prac_05/emails.py

```diff
@@ -0,0 +1,27 @@
+def extract_name(email):
+    # Extract the name from the email by splitting at '@' and taking the first part
+    return email.split('@')[0].title()
+
+
+def main():
+    user_info = {}
+
+    while True:
+        email = input("Email: ").strip()
+
+        if not email:
+            break
```

Comment:
Don't think we've used while True loops in this subject just yet, proceed with caution if using them for the assignment.

Reply:
I thought we had used True loops however in the case we haven't I have adjusted the file to use a variable called "is_true" and is set to true until where it used to have break in where it would then set it to false. This fixes that error however it still finishes the current loop but i think overall it is for the better